doc/openssl-c-indent.el

```diff
@@ -54,6 +54,7 @@
                 (arglist-close . c-lineup-arglist)           ; From "gnu" style
                 (inline-open . 0)                            ; From "gnu" style
                 (brace-list-open . +)                        ; From "gnu" style
+                (inextern-lang . 0)     ; Don't indent inside extern block
```

Comment:
Indentation of the comment?

Reply:
Doesn't fit within 80 columns.

Reply:
Didn't know that our coding guidelines also apply to lisp code. What about the curly brace rule? :-))